packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -23,6 +23,14 @@ export type BreadcrumbsProps = {
    * Defaults to the `SvgChevronRight` icon.
    */
   separator?: React.ReactNode;
+  /**
+   * If specified, this prop will be used to show a custom button when overflow happens,
+   * i.e. when there is not enough space to fit all the breadcrumbs.
+   *
+   * Expects a function that takes the number of items that are visible
+   * and returns the `ReactNode` to render.
+   */
```

Comment:
I think `@example` is needed here.

Reply:
I added an example for making the button direct to the previous breadcrumb, but it doesn't incorporate the parameter `visibleCount`. Do you think I should add the dropdown example in storybook here? I thought it was too long and had other components in it. I wanted my example to be useful to users though.

Reply:
Better show the actual usage because these examples don't show how real code should be.

Reply:
Is it ok if they're really long?

Reply:
We have kind of the same in `ButtonGroup` so it is not a problem as long it is helpful for the users.

Reply:
Updated examples